index.js

```diff
@@ -0,0 +1,60 @@
+let books = localStorage.getItem('books');
+books = JSON.parse(books) || [];
+localStorage.setItem('books', JSON.stringify(books));
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  static addBook = () => {
+    const ititle = document.getElementById('addTitle').value;
+    const iauthor = document.getElementById('addAuthor').value;
+    books.push({ title: ititle, author: iauthor });
+    localStorage.setItem('books', JSON.stringify(books));
+    this.renderBooks();
+  }
+
+  static removeBook = (e) => {
+    const button = e.target;
+    const title = button.id.split('_')[1];
+    books = books.filter((book) => book.title !== title);
```

Comment:
- [ ] **Required:** Kudos for a job well done. 👏. However, I noticed your remove functions don't behave properly. It removes multiple same entries.

![deletes dup](https://user-images.githubusercontent.com/27709832/165190935-2530281e-533d-49cc-b3a9-6ea084dc3f92.gif)

I would suggest you filter by something else eg. an id rather than author or title which you can have multiple of. You could introduce another attribute to your class. An id, index, etc.